docs/notes/2.23.x.md

```diff
@@ -63,6 +63,41 @@ The `output_path` field present on targets which can be packaged by `pants packa
 
 ### Backends
 
+#### NEW: nFPM
```

Comment:
Do you plan on having an `example-nfpm` repository users could look at? No need to develop it now or even reference it if you have one, but might be good to keep in mind as a follow-on.

Reply:
I hadn't thought about that. I think a section in the documentation might be useful as well, but I don't know when I can tackle something like that.